filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -154,7 +154,7 @@ VulkanDriver::VulkanDriver(VulkanPlatform* platform,
 
 #if ENABLE_VALIDATION
     if (createDebugReportCallback) {
-        static VkDebugReportCallbackCreateInfoEXT cbinfo = {
+        static constexpr VkDebugReportCallbackCreateInfoEXT cbinfo = {
```

Comment:
I would just use `const blah = ` and not `static`. I have no idea what the compiler will do, but it could run extra code to initialize this on first run.